lib/libvcc/vmodtool.py

```diff
@@ -803,6 +808,9 @@ def json(self, jl):
         ll.append(l2)
         self.fini.jsonproto(l2, self.fini.name)
 
+        if self.cli is not None:
+            self.cli.json(ll)
+
```

Comment:
Maybe make `$CLI` unconditional like `$INIT` and `$FINI` to simplify the libvcc side.